pkg/userdata/userdata.go

```diff
@@ -338,7 +338,18 @@ coreos:
         setup-compute \
         --cloud-provider={{ .CloudProviderName }}
 
+  - name: keto-tokens.service
```

Comment:
An seperate enhancement needed for keto-k8 to manage this step (via. docker run)... not in yet...
Currently keto-k8 will create and start kubelet.service and write out env file...